tests/unit/NameRequestInvalidErrorDialog.spec.ts

```diff
@@ -2,111 +2,128 @@ import Vue from 'vue'
 import Vuetify from 'vuetify'
 import { shallowMount } from '@vue/test-utils'
 import { NameRequestInvalidErrorDialog } from '@/components/dialogs'
-import { NameRequestStates } from '@/enums'
 
 Vue.use(Vuetify)
 
 const vuetify = new Vuetify({})
 
-describe('NameRequestInvalidErrorDialog - Displays Error/Warning messages', () => {
-  it('displays name request not found message', () => {
+describe('Name Request Invalid Error Dialog', () => {
+  it('displays Not Found message', () => {
     const wrapper = shallowMount(NameRequestInvalidErrorDialog,
       {
         vuetify,
         propsData: {
           dialog: true,
-          type: NameRequestStates.NOTFOUND
+          type: 'NOT_FOUND'
```

Comment:
If you use the actual string here then the unit test will catch the scenario where a developer changes the type value and forgets to update the unit test.